src/components/form/styles.ts

```diff
@@ -22,6 +22,11 @@ const FormModal: ThemeUICSSObject = {
   '[aria-label="panel-title"]': {
     textAlign: 'center',
   },
+  // id/className/customStyles props are only getting applied to checkbox and select <Field /> for whatever reason
+  // so this is a lovely little hack in the meantime to fix [DEVHUB-1451]
+  '[aria-label="institutionName"]': {
```

Comment:
not my proudest hack, might dig through Flora source to see if there props just aren't being applied to the `text-input` type of `<Field />` component